portfolio/src/main/webapp/script.js

```diff
@@ -75,6 +58,7 @@ function createCommentElement(comment) {
   });
   
   commentElement.appendChild(titleElement);
```

Comment:
Very good code structure where you defined the variable first and then used them in below. Also noticed you defined them in the order they are presented which is also great.

Reply:
+1. Very clean and easy to read!